simpleflow/swf/process/actor.py

```diff
@@ -214,10 +214,10 @@ def start(self):
         self.set_process_name()
         while self.is_alive:
             try:
-                task = self._poll(self.task_list, self.identity)
+                response = self._poll(self.task_list, self.identity)
```

Comment:
Yeah, rookie mistake! It's just a safeguard that should never trigger, but still.

I reinstalled Pycharm CE now, I'll try to see if I can get used to it. I'll fix the docs too, don't bother listing them here. Thanks for motivating me and making me better :D